src/public/Invoke-AzOpsGitPush.ps1

```diff
@@ -18,95 +18,97 @@ function Invoke-AzOpsGitPush {
             $skipPolicy = $false
         }
 
```

Comment:
FYI in #127  we need `git fetch origin` outside of the loop to ensure git locally has up to date branch from upstream